docs/HISTORY.txt

```diff
@@ -5,6 +5,7 @@ Changelog
 1.20.1 (unreleased)
 -------------------
 
+- Move title and show_title field into the coordinates schemata on MapBlock. [mathias.leimgruber]
```

Comment:
I'd try to be consistent with the format of the changelog and add a empty line between the list items.

Reply:
👍  done